pkg/distro/defs/rhel-10/distro.yaml

```diff
@@ -282,6 +282,16 @@
       - "mdadm"
       - "nss-softokn"
 
+  wsl_config: &wsl_config
+    config:
+      boot_systemd: true
+    distribution_config: &wsl_distribution_config
+      shortcut:
+        enabled: true
+        icon: /usr/share/pixmaps/fedora-logo.ico
```

Comment:
I know that this is a copy&paste, but referencing the Fedora logo in the RHEL definition does not look right.

Reply:
It is correct; all EL's put the logo in that path (weirdly so).